containers/popv/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM tensorflow/tensorflow:2.16.1
 
-RUN apt-get -y update && apt-get -y install python3-dev python3-pip g++ git wget && apt-get clean
```

Comment:
I assume the tensorflow docker already installs those dependencies you removed?

Reply:
Yes, it has python, pip, and wget